functions/datalayer/CategoryAddFunction.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
de category class in datalayer is voor alle functies.
deze class dus een generieke naam geven
CategoryDatabase of CategoryDB